modules/adding-cluster-notification-contacts.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * osd_cluster_admin/osd_logging/osd-accessing-the-service-logs.adoc
+// * rosa_cluster_admin/rosa_logging/rosa-accessing-the-service-logs.adoc
+
+:_content-type: PROCEDURE
+[id="adding-cluster-notification-contacts_{context}"]
+= Adding cluster notification contacts
+
+You can add notification contacts for your 
+ifdef::openshift-dedicated[]
+an {product-title} 
```

Comment:
Question, sir: Would it read better without the "an"?

Reply:
@GroceryBoyJr thank you for your review and suggestions!

Good catch, I have subsequently removed "an" from this line.